src/components/form/file_picker/_file_picker.scss

```
@@ -64,8 +64,7 @@
 /**
   * 1. Don't block the user from dropping files onto the filepicker.
   * 2. Ensure space for import icon, loading spinner, and clear button (only if it has files)
-  * 3. Delay focus gradient or else it will only partially transition while file chooser opens
-  * 4. Static height so that it doesn't shift its surrounding contents around
+  * 3. Static height so that it doesn't shift its surrounding contents around
```

lazy opinion: I would leave this as numbered 4 because there's a bunch of `/* 4 */` comments scattered through the file meant to reference this comment line - unless you want to update all of the above said references to `/* 3 */
```suggestion
  * 4. Static height so that it doesn't shift its surrounding contents around
```